vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/GeneratedVaadinTextField.java

```diff
@@ -173,31 +173,7 @@
 @JsModule("@vaadin/text-field/src/vaadin-text-field.js")
 public abstract class GeneratedVaadinTextField<R extends GeneratedVaadinTextField<R, T>, T>
         extends AbstractSinglePropertyField<R, T>
-        implements HasStyle, Focusable<R>, HasTheme {
```

Comment:
All text fields extend from this class, so all of them inherited the `addThemeVariants` / `removeThemeVariants` methods. I could have just added the new interface here, but I felt it's better to explicitly add the interface to each field, and cover that with tests.